recipe/meta.yaml

```diff
@@ -21,8 +21,9 @@ requirements:
     - setuptools_scm
   run:
     - python >=3.9
-    - pydm >=1.16.0
+    - pydm >=1.9.0
     - qtpy
+    - pyqt >=5
```

Comment:
I'm not sure I necessarily agree with these dependency changes
I think the pydm pin update will probably have little impact
I think this package doesn't actually rely on pyqt directly, so if we need a "does not work with pyqt4" sort of deal then this should be a run constraint instead